cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -233,6 +239,12 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             );
         },
 
+        callAnalytics: function (event) {
+            event.preventDefault();
+            analytics.track(event.currentTarget.className + " clicked.");
```

Comment:
Regardless, @sandroroux we'll probably want to use something other than the element class name in the event name. Maybe add an if block here that checks the `event.currentTarget.className` and then sets an `eventName` variable to "Highlights Modal Save Clicked" or "Hightlights Modal Cancel Clicked".

Reply:
@thallada, I rewrote this to display `event.target.innerHTML` in the event label. The label will say "[Cancel|Save] clicked in Highlights Modal".

The printout will break if someone ever changes the `innerHTML` of the buttons. Is it worth locking down the button contents? That is, assert them in a test?